File: Python Word Examples/FieldsApiDemo/updateField.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.fields_api.FieldsApi(configuration)

file_name = "UpdateField.docx"
extra = {'folder': 'input',
         'dest_file_path': 'Fields/UpdateField.docx'}
result = api.update_field(file_name, **extra)

File: JavaScript Word Examples/FieldsApiDemo/updateField.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.FieldsApi(apiClient);
    
    var fileName = "Fields.docx";
    var destFilePath = "output/Fields/UpdateField.docx";
    var opts = { 'folder': 'Fields' };
    instance.updateField(fileName, destFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();